controllers/feeds/getFeedChat.js

```diff
@@ -39,7 +39,7 @@ const mappingFeed = (req, feeds) => {
   return mapping;
 };
 
-const getDetail = (req, b, id) => {
+const getDetail = (req, b, id, anonymousId) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getDetail` now has a new parameter `anonymousId`. This change seems to be consistent with the PR title and description. However, it's important to ensure that all calls to this function are updated to pass this new argument.
```diff
- const getDetail = (req, b, id) => {
+ const getDetail = (req, b, id, anonymousId) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->